srf_generation/input_file_generation/realisation_to_srf.py

```diff
@@ -836,22 +1038,34 @@ def generate_sim_params_yaml(
 
     logger.debug(f"Processed sim params: {sim_params}")
     logger.debug(f"Saving sim params to {sim_params_file}")
-    with open(sim_params_file, "w") as spf:
+    with open(sim_params_file, "w", encoding="utf-8") as spf:
         yaml.dump(sim_params, spf)
-    logger.debug(f"Sim params saved")
+    logger.debug("Sim params saved")
 
 
-def load_args():
+def load_args() -> argparse.Namespace:
```

Comment:
Personally I would get rid of this function and just move the code into main. Its 3 lines of code that are never re-used. But thats just my opinion, happy for you to leave as is.

Reply:
I'll leave this function (for now). When I come back and rewrite the SRF gen code after the type-5 work I'll rewrite it without the load_args function. I agree that it is too trivial to warrant it's own function.